docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,431 @@
+
+<Figure align="left" bordered caption="Enable the Smart Card service">
+  ![Enable Smartcard](../../../img/desktop-access/smartcard.png)
+</Figure>
+
+<Admonition type="note" title="gpupdate.exe">
+  You will be modifying GPOs, and sometimes GPO modifications can take some time
+  to propagate to all hosts. You can force your changes to take effect
+  immediately on your current host at any time by opening a PowerShell prompt
+  and running `gpupdate.exe /force` (though their effects may still take time to
+  propagate to other machines on the domain).
+</Admonition>
+
+### Publish the Teleport CA
+
+This step enables the domain controllers to trust the Teleport CA, which will
+allow smart card logins via Teleport to succeed.
+
+On a machine that is joined to your domain and logged in as an account in the
+`Domain Administrators` group, run the two commands below at a PowerShell prompt
+to publish the Teleport CA to your Active Directory domain (using the path
+to the exported Teleport `db-ca.cer` file that you copied above):
+
+```powershell
+certutil –dspublish –f <PathToCertFile.cer> RootCA
+certutil -dspublish -f <PathToCertFile.cert> NTAuthCA
+```
+
+### Publish the Teleport CRL
+
+On the same machine, run the command below at a PowerShell promt to publish the
+Teleport CRL to your Active Directory domain (using the path to the exported
+`db-ca.crl` file that you copied above).
+
+```powershell
+certutil -dspublish -f <PathToCRLFile.crl> TeleportDB
+```
+
+<Admonition type="tip">
+  To avoid waiting until the certificate propagates, you can force the CA
+  retrieval from LDAP after importing the CA and CRL with the command:
+
+  ```powershell
+  certutil -pulse
+  ```
+</Admonition>
+
+## Step 3/7. Export the LDAP CA certificate
+
+Teleport uses LDAPS to authenticate users, which requires specifying the LDAP CA
+certificate on the database configuration. To ensure that Teleport trusts the
+certificate sent by the server during the initial SSL connection, you must export
+the certificate from your AD. You can export the certificate by running the
+following PowerShell script on your Windows instance:
+
+```powershell
+$WindowsDERFile = $env:TEMP + "\windows.der"
+$WindowsPEMFile = $env:TEMP + "\windows.pem"
+certutil "-ca.cert" $WindowsDERFile 
+certutil -encode $WindowsDERFile $WindowsPEMFile
+
+$CA_CERT_PEM = Get-Content -Path $WindowsPEMFile
+Write-Output $CA_CERT_PEM
+
+Remove-Item $WindowsDERFile -Recurse
+Remove-Item $WindowsPEMFile -Recurse
+```
+
+The script will write the LDAP CA contents in PEM format to the terminal, and
+from there, you can copy and use it on your database configuration.
+
+## Step 4/7. Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Teleport Database Service must run on a Linux server with the `kinit` command
```

Comment:
I think we should indicate in the Prerequisites that the host where you'll run the Database Service needs to have `kinit` installed. That way, the user can be confident that their environment is ready for this guide after finishing that section